### DISCUSSION TITLE
Consider introspecting UIContextMenuInteraction

### DISCUSSION THREAD

**Discussion by mycroftcanner** [siteline/SwiftUI-Introspect#250] [Ideas]
This would be useful because there is no way to know when the context menu appears.

**Comment by GeorgeElsham** [comment]
Is there a method on `UIContextMenuInteraction` you can call to even do that? Unfortunately, if you can only detect that by overriding a method like `didMove(to:)`, it's likely not possible.

**Comment by davdroman** [comment]
I concur with @GeorgeElsham. There isn't much one can do because context menus don't have a publicly available UIView-based class. This is similar to how tab bar items are represented by an abstract `UITabBarItem` class. So even if we could pluck out the menu view casted as a `UIView`, it wouldn't be of much use because it lacks all relevant information.
